AppCenter/AppCenterTests/MSJwtClaimsTests.m

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSJwtClaims.h"
+#import "MSTestFrameworks.h"
+
+@interface MSJwtClaimsTests : XCTestCase
+
+@end
+
+@implementation MSJwtClaimsTests
+
+static NSString *const kMSJwtFormat = @"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.%@";
+
+- (void)testGetValidJwt {
+
+  // If
+  NSString *userId = @"some_user_id";
+  int expiration = 1426420800;
```

Comment:
Let's not use a magic number. Maybe leave a comment or rename the variable.

Reply:
It doesn't actually need to be a specific value.  Would you prefer to just use 0 for everything?

Reply:
0 works for me.